sdk/maps/azure-maps-render/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 1.0.0-beta.1 (2022-07-22)
```

Comment:
Can we change the release date to 2022-09-06, which is the planned release date we want. Same as Search and Route